### PR TITLE
Update the default cluster location

### PR DESCRIPTION
**Pull request by lohi212**
## What is this PR for?

https://www.notion.so/appbase/Update-the-default-cluster-location-6f16e1a1525e42bfa3d18f81a2a42047

## How have you tested this PR?

Loom: https://www.loom.com/share/33a9ef3e9b6b4350ba4ca5a6e1aae488

### CODE REVIEW

**Comment by siddharthlatest** [comment]
@lohi212 Remove the decimal from the ping time: https://i.imgur.com/ZM475xV.png. The value should be a whole number.

**Comment by lohi212** [comment]
https://www.loom.com/share/9534f70841964c00b32e20f9ca3fa366

File: src/pages/ClusterPage/new.js [hunk 1]
```diff
@@ -525,36 +623,52 @@ class NewCluster extends Component {
...
+								<span> {regionValue.name} </span>
+							</li>
+						);
+					})}
+				</div>
+				{pingTime ? (
+					<div className="ping-time-container">
+						Expected ping latency for{' '}
+						{regions[provider][this.state.region].name} (
+						{this.state.region}) is: {pingTime}ms
```
Comment (siddharthlatest) [inline src/pages/ClusterPage/new.js]:
```suggestion
						{this.state.region}) from your location is: {pingTime}ms
```

File: src/pages/ClusterPage/NewMyCluster.js [hunk 2]
```diff
@@ -280,36 +287,52 @@ class NewMyCluster extends Component {
...
+								<span>{regionValue.name}</span>
+							</li>
+						);
+					})}
+				</div>
+				{pingTime ? (
+					<div className="ping-time-container">
+						Expected ping latency for{' '}
+						{regions[provider][this.state.region].name} (
+						{this.state.region}) is: {pingTime}ms
```
Comment (siddharthlatest) [inline src/pages/ClusterPage/NewMyCluster.js]:
```suggestion
						{this.state.region}) from your location is: {pingTime}ms
```

**Review by siddharthlatest — APPROVED** [review]

**Review by siddharthlatest — CHANGES_REQUESTED** [review]
**Issue**:

- [ ] There should be a live value that is displayed as the API calls are made before the pulse. The pulse animation is to indicate that the value isn't final yet. Currently, no time is being shown.

**Pull request merged**